frontend/src/components/TaskMapTask.tsx

```diff
@@ -26,7 +26,10 @@ export interface TaskProps {
   setSelectedTask?: any;
   checked: { from: boolean; to: boolean };
   disableDragging: boolean;
+  dropDisabled?: boolean;
+  setGroupDraggable: any;
```

Comment:
```suggestion
  setGroupDraggable: (_: boolean) => unknown;
```

Could be a suitable type, since that is what it is used for